sqlx-core/src/mysql/options.rs

```diff
@@ -231,6 +244,10 @@ impl FromStr for MySqlConnectOptions {
                     options = options.ssl_ca(&*value);
                 }
 
+                "statement-cache-size" => {
```

Comment:
Where should I document this?

Reply:
We should probably have something like the following at the type-level documentation for `MySqlConnectOptions`, `PgConnectOptions`, etc.:

---

#### Parameters

| Parameter | Default | Description |
| --- | --- | --- |
| `ssl_mode` or `sslmode` | `prefer` | Determines whether or with what priority a secure SSL TCP/IP connection will be negotiated. See `PgSslMode`. |
| `statement_cache_size` | `100` | The maximum size of the statement cache. Set to 0 to disable statement caching (useful for environments that do not support this, such as proxies). |